keyboards/squarebuilt/keymaps/default/keymap.c

```diff
@@ -0,0 +1,94 @@
+/* Copyright 2019 LanceMercado
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 16
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+  QMKBEST = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+  QMKURL
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_QWERTY] = LAYOUT( \
+  KC_Q,    KC_W,    KC_E,          KC_R,               KC_T,              KC_Y,          KC_U,               KC_I,          KC_O,    KC_P,  \
+  KC_A,    KC_S,    KC_D,          KC_F,               KC_G,              KC_H,          KC_J,               KC_K,          KC_L,    KC_SCLN, \
+  KC_Z,    KC_X,    KC_C,          KC_V,               KC_B,              KC_N,          KC_M,               KC_COMM,       KC_DOT,  KC_SLSH, \
+  KC_ESC,  KC_LGUI, ALT_T(KC_ENT), LT(_LOWER, KC_DEL), CTL_T(KC_BSPC),    SFT_T(KC_SPC), LT(_RAISE, KC_TAB), OSM(MOD_RSFT), KC_PSCR, KC_ENT \
```

Comment:
It look like you're using a hacky way to do tri-layers here.  I would recommend against that. 

Specifically, remove the `MO(x)` keycodes in the raise and lower layer and add the following to your keymap.c file: 

```c
layer_state_t layer_state_set_user(layer_state_t state) {
  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
}
```

https://docs.qmk.fm/#/ref_functions?id=olkb-tri-layers